### PR TITLE
add: MikroTik generator config and output

### PR DESCRIPTION
**Pull request by lukassup**

### CODE REVIEW

File: generator/generator.yml [hunk 1]
```diff
@@ -218,3 +218,14 @@ modules:
...
+# MiktoTik
+#
+# http://download2.mikrotik.com/Mikrotik.mib
+#
+  mikrotik:
+    walk:
+      - sysUpTime
+      - interfaces
+      - ifXTable
+      - 1.3.6.1.4.1.14988.1.1
```
Comment (brian-brazil) [inline generator/generator.yml]: What's this object's name?

Comment (lukassup) [reply generator/generator.yml]: `snmptranslate` gives me this:
```
.iso.org.dod.internet.private.enterprises.mikrotik.mikrotikExperimentalModule.mtXRouterOs
```

Comment (brian-brazil) [reply generator/generator.yml]: We don't usually pull in entire MIBs. Can you be more specific about which parts of it you want?

**Comment by SuperQ** [comment]
Based on what I see in the MIB, there whole table is probably a bad idea.  There are some things indexed by full MAC address.  I think these walks should be safe / useful.

```
- mtxrWlStatTable
- mtxrWlApTable
- mtxrDnStatTable
- mtxrInterfaceStatsTable
- mtxrHealth
- mtxrLicense
- mtxrDHCPLeaseCount
```

**Comment by brian-brazil** [comment]
This has gone stale, if you're still interested in working on this let us know.

**Pull request closed without merging**